staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -349,9 +349,9 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	StorageVersionHash: {Default: true, PreRelease: featuregate.Beta},
 
-	StrictCostEnforcementForVAP: {Default: false, PreRelease: featuregate.Beta},
+	StrictCostEnforcementForVAP: {Default: true, PreRelease: featuregate.GA, LockToDefault: true},
```

Comment:
The previous PR is merged, so you'll need to update versioned_kube_features here:

https://github.com/kubernetes/kubernetes/blob/master/pkg/features/versioned_kube_features.go#L252-L254

and https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/features/kube_features.go#L405

Just add a new line for the GA feature gate status so we can preserve beta transition information.

Reply:
Thank you. I have updated the PR to pick up the versioned fg. Currently rebased on https://github.com/kubernetes/kubernetes/pull/127643 until it's merged.